server/src/main/java/io/crate/auth/ClientCertAuth.java

```diff
@@ -33,11 +33,22 @@
 
 public class ClientCertAuth implements AuthenticationMethod {
 
-    static final String NAME = "cert";
+    public static final String NAME = "cert";
     private final UserLookup userLookup;
 
+    // SSL is normally required for cert method,
+    // this flag represents an HBA config entry corresponding
+    // to the concrete instance of CertAuth and letting user to downgrade to non-SSL.
```

Comment:
Maybe move to the getter method as regular java docstring - so that the information is visible in editors/IDEs when they call the method